sky/clouds/service_catalog/data_fetchers/fetch_azure.py

```diff
@@ -36,6 +36,34 @@
 
 SINGLE_THREADED = False
 
+# Family name to SkyPilot GPU name mapping.
+#
+# When adding a new accelerator:
+# - The instance type is typically already fetched, but we need to find the
+#   family name and add it to this mapping.
+# - To inspect family names returned by Azure API, check the dataframes in
+#   get_all_regions_instance_types_df().
+FAMILY_NAME_TO_SKYPILOT_GPU_NAME = {
+    'standardNCFamily': 'K80',
+    'standardNCSv2Family': 'P100',
+    'standardNCSv3Family': 'V100',
+    'standardNCPromoFamily': 'K80',
+    'StandardNCASv3_T4Family': 'T4',
+    'standardNDSv2Family': 'V100-32GB',
+    'StandardNCADSA100v4Family': 'A100-80GB',
+    'standardNDAMSv4_A100Family': 'A100-80GB',
+    'StandardNDASv4_A100Family': 'A100',
+    'standardNVFamily': 'M60',
+    'standardNVSv2Family': 'M60',
+    'standardNVSv3Family': 'M60',
+    'standardNVPromoFamily': 'M60',
+    'standardNVSv4Family': 'Radeon MI25',
+    'standardNDSFamily': 'P40',
+    'StandardNVADSA10v5Family': 'A10',
+    'StandardNCadsH100v5Family': 'H100',
+    'standardNDSH100v5Family': 'H100',
```

Comment:
Weirdly, I don't see this family in `az vm list-skus --all --resource-type virtualMachines -l southcentralus | grep v5`. How do we get this family?

Reply:
Running this on my end got
```
...
    "family": "StandardNCadsH100v5Family",
    "name": "Standard_NC40ads_H100_v5",
    "size": "NC40ads_H100_v5",
    "family": "StandardNCadsH100v5Family",
    "name": "Standard_NC80adis_H100_v5",
    "size": "NC80adis_H100_v5",
...
```
My `az account subscription list` shows subscripts `*7` and `*a`.

Reply:
It was obtained by inspecting an intermediate dataframe. Also seen in:
```
» az vm list-skus --all --resource-type virtualMachines  | grep -i h100
```
i.e., with the location removed.

Reply:
Oops, sorry. I forgot that location argument. Just removed the location and see the family. Thanks for the explanation.